src/style.css

```diff
@@ -0,0 +1,41 @@
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__link {
+  display: block;
+  line-height: 40px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__buy .card__link:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border-radius: 5px;
+  transition: background-color 0.2s, color 0.2s;
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
This is a extra class, but if you need it you should add that part of code in file `stars.css`

Reply:
fixed